releases/v2.1.0-beta.20181015.md

```diff
@@ -0,0 +1,98 @@
+---
+title: What&#39;s New in v2.1.0-beta.20181015
+toc: true
+summary: Additions and changes in CockroachDB version v2.1.0-beta.20181015 since version v2.1.0-beta.20181008
+---
+
+## October 15, 2018
+
+Get future release notes emailed to you:
+
+<div class="hubspot-install-form install-form-1 clearfix">
+    <script>
+        hbspt.forms.create({
+            css: '',
+            cssClass: 'install-form',
+            portalId: '1753393',
+            formId: '39686297-81d2-45e7-a73f-55a596a8d5ff',
+            formInstanceId: 1,
+            target: '.install-form-1'
+        });
+    </script>
+</div>
+
+### Downloads
+
+<div id="os-tabs" class="clearfix">
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-beta.20181015.darwin-10.9-amd64.tgz"><button id="mac" data-eventcategory="mac-binary-release-notes">Mac</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-beta.20181015.linux-amd64.tgz"><button id="linux" data-eventcategory="linux-binary-release-notes">Linux</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-beta.20181015.windows-6.2-amd64.zip"><button id="windows" data-eventcategory="windows-binary-release-notes">Windows</button></a>
+    <a href="https://binaries.cockroachdb.com/cockroach-v2.1.0-beta.20181015.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### SQL language changes
+
+- The output of [`SHOW JOBS`](../v2.1/show-jobs.html) now reports ongoing jobs first in start time order, followed by completed jobs in finished time order. [#31005][#31005] {% comment %}doc{% endcomment %}
+- CockroachDB now supports more customizations from PostgreSQL client drivers when initially setting up the client connection. [#31021][#31021]
+- Columns that are part of a table's [`PRIMARY KEY`](../v2.1/primary-key.html) can no longer be specified as [`STORING` columns](../v2.1/create-index.html#store-columns) in secondary indexes on the table. [#31032][#31032] {% comment %}doc{% endcomment %}
```

Comment:
is `{% comment %}doc{% end comment %}` supposed to be here?

Reply:
Those are there just so I can generate docs issues from the notes.